ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -425,6 +428,7 @@ private class JdbcLedgerDao(
       offset: Offset,
       transaction: CommittedTransaction,
       divulged: Iterable[DivulgedContract],
+      blindingInfo: Option[BlindingInfo],
```

Comment:
How are we going to use the `blindingInfo` here?

Reply:
This storage reduction story, as a whole, will move blinding info computation from the read to the write path and will optimize its storage layout by making it explicit.

As a preparation to support the new format without dropping the old one, `TransactionCommitter` (KV) and `SqlLedger` (classic) will pre-compute this field from the transaction as it is done today by the indexer on the read path (`Blinding.blind`), it will be stored on-ledger and then `KeyValueConsumption` will read it back. Also, the indexer will prefer the pre-computed info if available.

As the very last step of the whole story, `TransactionCommitter` will prune `Fetch` and `LookupByKey` transaction nodes, activating the storage reduction.